R/missingLevelsTrain_lm.R

```diff
@@ -0,0 +1,42 @@
+missingLevelsTrain.lm = function(model, test.data) {
```

Comment:
Maybe there are more gentle ways to write this but it works and is quite robust (at least I hope so :smile:). 
I do not really want to change anything here because (1) it works, (2) is well documented and (3) traceable. Do you really see an important need to change the complete approach here?